File: langs/python/utils/from_file.py
class FromFile:
  def set(self,ss):
    if len(ss) == 0: self.empty = True; self.ss =""
    else: self.empty = False; self.ss = ss
  def all(self)  : return self.ss
  def head(self) : return self.ss[0]
  def tails(self): return [self.ss[i] for i in range(1,len(self.ss))]

def ffUser(path):
  ff = FromFile()
  with open(path, mode='r', encoding='utf-8') as f:
    ff.set([s.strip() for s in f.readlines() if s != ""])
  return ff
if __name__ == "__main__":
  issue1 = "a:/pj/sc/lib/rs/issues/issue1.txt"
  res = ffUser(issue1)
  print(res.head())

File: langs/ruby/utils/from_file.rb
#encoding: UTF-8

class FromFile
  attr_reader :empty, :ss
  def set(ss) if ss.size == 0 then @empty = true; @ss = ''; else @empty = false; @ss = ss end end
  def all() @ss end
end

def ffUser(path) ff = FromFile.new; f = File.open path,"r:utf-8"; ff.set f.readlines.each{|l| l.rstrip!}; f.close; ff; end

if __FILE__ == $0 then
  issue = "a:/pj/sc/lib/rs/issues/issue1.txt"
  res = ffUser(issue)
  puts res.all
end